packages/Pagination/src/Pagination.js

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import PropTypes from "prop-types";
+// import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import ArrowItem from "./components/ArrowItem/ArrowItem";
+import CurrentPageItem from "./components/CurrentPageItem/CurrentPageItem";
+import ElipsisItem from "./components/EllipsisItem/ElipsisItem";
+import PageItem from "./components/PageItem/PageItem";
+import Divider from "./components/Divider";
+import * as sc from "./Pagination.styles";
+
+const propTypes = {
+  /** The number of current active page */
+  currentPage: PropTypes.number.isRequired,
+  /** The number of other pages that will be visible around the current/active page (not hidden by elipsis). */
+  pagesOnEachSide: PropTypes.number,
+  /** Callback to be executed when current page is changed. */
+  onClick: PropTypes.func.isRequired,
```

Comment:
ohhh I thought its was a onClick, because was named onClickPage, but after reading the comment seems is an onChange and not a onClick, good call @allison-c can we rename it @KaanDarcey ?